examples/worker-cert.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "giantswarm.io/v1"
+kind: Certificate
+metadata:
+  name: "uo91f-worker"
+  namespace: "uo91f"
+spec:
+  clusterID: "uo91f"
+  clusterComponent: "worker"
+  commonName: "api.uo91f.g8s.eu-west-1.aws.test.private.giantswarm.io"
+  altNames:
+  - "uo91f.g8s.eu-west-1.aws.test.private.giantswarm.io"
+  - "k8s-master-vm"
```

Comment:
why does the worker cert need these alt names? these are the api server ANs aren't they?